Frameworks/include/CollectionHelpers.h

```diff
@@ -81,7 +81,7 @@ HRESULT WRLToNSCollection<ABI::Windows::Foundation::Collections::IMap<HSTRING, H
         });
     RETURN_IF_FAILED(hr);
 
-    *pDictionary = collection;
+    *pDictionary = [collection autorelease];
```

Comment:
Agreeed. if we hit an exception before here, there will be sadness. 

---
In reply to: [107801632](https://github.com/Microsoft/WinObjC/pull/2312#discussion_r107801632) [](ancestors = 107801632,107799430)